test/test_torch.py

```diff
@@ -5322,6 +5322,18 @@ def add_neg_dim_tests():
 class TestTorchDeviceType(TestCase):
     exact_dtype = True
 
+    # Verifies that the inplace dunders (like idiv) actually are in place
+    @onlyOnCPUAndCUDA
+    def test_inplace_dunders(self, device):
+        t = torch.randn((1,), device=device)
```

Comment:
how about testing inplace and, or, and shift operators.

Reply:
Good idea. I'll put those in a follow-up and like extend our tensor method tests to include these dunders, too.